src/internal/ResourceCall.js

```diff
@@ -63,21 +93,47 @@ class ResourceCall extends Smart {
     return parsedPayload;
   }
 
+  _requestOptions(payload) {
+    const { method, headers, cors } = this.context;
+
+    const options = {
+      method,
+      credentials: 'same-origin',
+      cors: cors ? 'cors' : 'no-cors',
+      headers: new Headers(headers),
+    };
+
```

Comment:
no line breaks within methods! 
if you feel like you need it for better readability, then split it in multiple submethods